extensions/machine-learning-services/src/common/apiWrapper.ts

```diff
@@ -105,4 +105,24 @@ export class ApiWrapper {
 	public showQuickPick<T extends vscode.QuickPickItem>(items: T[] | Thenable<T[]>, options?: vscode.QuickPickOptions, token?: vscode.CancellationToken): Thenable<T | undefined> {
 		return vscode.window.showQuickPick(items, options, token);
 	}
+
+	public listDatabases(connectionId: string): Thenable<string[]> {
+		return azdata.connection.listDatabases(connectionId);
+	}
+
+	public openTextDocument(options?: { language?: string; content?: string; }): Thenable<vscode.TextDocument> {
```

Comment:
nit: indenation seems bit off

Reply:
not sure which part you mean

---
In reply to: [389165915](https://github.com/microsoft/azuredatastudio/pull/9450#discussion_r389165915) [](ancestors = 389165915)

Reply:
now showing now for some reason.. ignore this..